examples/Best Practice/common_utilities/test_plan.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
Let's add a separate example for extract_driver_info, that has a few interconnected drivers (e.g client -> app -> server), and then from the logged driver info, we can figure out how they are connected.

Reply:
Yeah, sorry, this is actually incomplete, I did not even add anything to it, just moved the example.